.github/workflows/build-test-dev.yml

```diff
@@ -6,7 +6,7 @@ on:
       - main
 
 env:
-  VERSION: 1.0.3
+  VERSION: 1.2.0-dev
```

Comment:
what is the criteria for selecting the next version?

Reply:
This is just for dev testing.